tests/functional/smoke/full-compaction.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
+
+#doc: simple transaction test on a KVDB kvs
+
+. common.subr
+
+trap cleanup EXIT
+kvdb_create
+
+kvs=$(kvs_create smoke-0)
+
+cmd kmt "$home" "$kvs" -b -i128m -l16 -s1 -t10 -j64
+cmd hse kvdb compact -f -t 1200 $home
+
+cn_metrics "$home" "$kvs" | grep -q -P "^t\s+1\s+1\b"
+[[ $? == 0 ]] || err "Tree is not fully compacted"
+
```

Comment:
Looks like you have an extra newline in this file at the end.

Reply:
We shouldn't be so militant as to not allow at least one "extra" newline at the end.  Nice for when you cat a file, provides a little breathing room between files and/or your prompt...

Reply:
Not quite sure what you mean. My editor has support for not butting the last line up against the bottom of the buffer as do Vim and its derivatives. I would assume emacs has a similar feature.

Reply:
cat

Reply:
cat *.c

Point is, sometimes extra blank lines improve readability, it's stupid to allow some hard, fixed, uncaring, unknowing mechinistic rule to override a human's best judgement...

Reply:
```
$ for f in *.c; do cat $f && echo; done
```

Source code shouldn't be stylized for text-based tooling in my opinion. We have Tree Sitter and Language Servers that are much better at handling source code than grep, sed, awk, cat, etc.,  ever were.

Either way it isn't like I am blocking this PR from merging because of this new line. But this is annoying for other people like me who configure their editors to remove extra new lines at the end of the file, and it will just be pointless noise in the next diff.

> override a human's best judgement...

What do you do when judgement differs between contributors?

Reply:
Try `head -n-0 -v *.c` to view multiple files on stdout.